hookman/hookman_utils.py

```diff
@@ -0,0 +1,93 @@
+import ctypes
+import os
+from collections import OrderedDict
+from pathlib import Path
+from typing import List, Optional, Union
+
+
+def load_plugin_config_file(hook_config_file: str) -> OrderedDict:
+    """
+    Load the content of the plugin.yaml file.
+    Noticed that the entry shared_lib will be update according to the operating system
+
+    Obs.: This method receives a str instead of Path because when the ZipFile is loaded (on install_plugin)
```

Comment:
I made this method receive a string because on method `install_plugin`  (file `hooks.py` ) I'm reading the content of the zip file before the extraction (to validate if everything is fine).

Reply:
- [x] Fixed